projects/1.20/assets/mopeds/moped/lang/zh_cn.json

```
@@ -0,0 +1,56 @@
+{
+  "advancements.booster_adv.descr": "你制作了助力车助推器",
+  "advancements.booster_adv.title": "升级，伙计们，快升级",
+  "advancements.copper_moped_adv.descr": "你制作了铜制助力车",
+  "advancements.copper_moped_adv.title": "马儿向前冲",
+  "advancements.copper_moped_root.descr": "你已经安装了助力车模组，希望你也安装了Create模组",
+  "advancements.copper_moped_root.title": "想去兜个风吗？",
+  "advancements.fully_upgraded_moped.descr": "给一辆助力车装载3个助推器并骑上它",
+  "advancements.fully_upgraded_moped.title": "这才叫赛车！",
+  "advancements.moped_honk.descr": "助力车鸣笛",
+  "advancements.moped_honk.title": "快躲开！！！！！",
+  "advancements.moped_key_adv.descr": "你需要一把钥匙来打开这辆助力车的油箱",
+  "advancements.moped_key_adv.title": "我怎么给这东西加油？",
+  "advancements.sign_adv.descr": "放置任何助力车模组中的标志",
+  "advancements.sign_adv.title": "这儿有停车位！",
+  "advancements.tiny_copper_moped_adv.descr": "你制作了微型铜制助力车",
+  "advancements.tiny_copper_moped_adv.title": "马儿加油（Pony Go Vroom）",
```

```suggestion
  "advancements.tiny_copper_moped_adv.title": "极速马儿",
```
原文就蛮口语了（）
虽然这个也不咋口语就是